src/api/app/views/webui2/webui/projects/meta/_breadcrumb_items.html.haml

```diff
@@ -0,0 +1,24 @@
+= render partial: 'webui/main/breadcrumb_items'
```

Comment:
These are the breadcrumbs from project, so it needs to be `= render partial: 'webui/project/breadcrumb_items'`